python/MRzeroCore/phantom/brainweb/__init__.py

```diff
@@ -141,6 +141,11 @@ def noise() -> np.ndarray:
                 for alias in brainweb_data["download-aliases"][tissue]
             ])
             tissue_map = downsample(tissue_map, config_data["downsample"])
+            tiusse_mask = tissue_map > 0.2
```

Comment:
Here the masks are binary, but the input segmentation data is float because voxels can be mixtures of multiple tissues at the borders of the tissues.

I don't know the exact application, but it might be more suitable to keep the masks in the [0, 1] range thoughout the code and as float32

Reply:
I understand, and realize that masks can be converted to binary later on at the user end with their preferred thresholding and use. In this case, there is no reason to add a new `tissue_mask` item to dict and we can use the `tissue_map` ( the tissue segmentation map ranging [0,1] which is already part of the data, so I just added it into the `voxel_grid_phantom`  subsequent build process. I have already made these changes. Please have a look and let me know if there are any other changes that need to be made.